data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,409 @@ mission "Ringworld Debris: Quarg"
+					goto explore
+				`	(Leave and return to Aventine.)`
+			`	As you are preparing to jump away, your eye is caught by an artificial structure near the edge of the iridium storage. The first thing you notice is an array of extremely high pylons put into two parallel lines; then, when you are able to get a better angle of view, you discover a large cupola towering all over the area. Although understanding its scale without a reference point is hard, you estimate it must be at least a kilometer in diameter.`
+				goto image
+			label explore
+			`	While you are setting up a new route to pass above as much terrain as possible, your eye is caught by an artificial structure near the edge of the iridium storage. The first thing you notice is an array of extremely high pylons put into two parallel lines; then, when you are able to get a better angle of view, you discover a large cupola towering all over the area. Although understanding its scale without a reference point is hard, you estimate it must be at least half a kilometer in diameter.`
+			label image
+			scene "scene/asteroid scene"
+			choice
+				`	(Go near the structure, regardless of the risk.)`
+					goto near
+				`	(Collect measurements at a safe distance.)`
+			`	The data that Dusk's scanners will gather will not be perfect, but it is better than having to risk getting close to something you know so little about. After a few minutes, the scan completes, and you are ready to return to Aventine.`
+				accept
+			label near
+			apply
+				"exploration interest" += 1
+			`	You carefully position your ship above the structure and activate the sensors. As the scan goes by, you monitor the position of the Ka'sei fleet. You notice that some of the Fetri'sei nearest to you stopped their standard activities and are just standing by, perhaps observing your actions the same way you are looking at theirs. Luckily, after a few minutes the sensors complete their analysis of the structure and you are ready to return to Aventine.`
+				accept
+	on complete
+		dialog
+			`Near the landing area, you discuss with Dusk what you have found next to the iridium storage. "This is fantastic! It is so much more than I hoped to find with this expedition. Give me a few hours to analyze the data and meet me in the spaceport."`
+
+mission "Ka'het: Patir Mystery 4"
+	name "Visit the asteroid"
+	description "Bring Dusk to <system> to visit the ancient Builder settlement."
+	source "Aventine"
+	destination "Builder Settlement"
+	passengers 1
+	to offer
+		has "Ka'het: Patir Mystery 3: done"
+	on offer
+		conversation
+			`You meet Dusk again in the cafeteria. He has brought a touchpad with him this time as well. "Thank you again for your help, <first>. There is a lot we can learn about the Builders with this discovery." He shows you a black and white image of a building inside a large semi-sphere. The complex appears to be made of interconnected cylinders, presumably habitation modules.`
```

Comment:
```suggestion
			`You meet Dusk again in the cafeteria with his touchpad, as usual. "Thank you again for your help, <first>. There is a lot we can learn about the Builders with this discovery." He shows you a black and white image of a building inside a hemispherical dome. The complex appears to be made of interconnected cylinders, presumably habitation modules.`
```

(he brought one last time too, so the fact that he did this time is not unusual)